### DISCUSSION TITLE
CypherpunkPay

### DISCUSSION THREAD

**Discussion by nyxnor** [raspiblitz/raspiblitz#3926] [Feature / App Ideas]
[Git repo](https://github.com/CypherpunkPay/CypherpunkPay)
[Deb repo for apt](https://deb.cypherpunkpay.org/)
[Demo](https://store.cypherpunkpay.org/cypherpunkpay/dummystore/)
[Docs](https://cypherpunkpay.org/)

This is more like a lightweight BTCPaysServer (STORE) that is a Linux Daemon.

**Comment by rootzoll** [comment]
Just to collect the idea or any commitment to implement?

**Comment by nyxnor** [comment]
just collecting the idea for now, but this one seems easy as it is already a daemon if someone wants to take it.
